### ISSUE TITLE
Can only configure Slowmode/NSFW for channels of types...

### ISSUE THREAD

**Issue by IBPrivate** [discord-jda/JDA#2537]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [X] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

### Expected Behaviour

```
java.lang.IllegalArgumentException: Can only configure Slowmode for channels of types TEXT, GUILD_NEWS_THREAD, GUILD_PUBLIC_THREAD, GUILD_PRIVATE_THREAD, FORUM
java.lang.IllegalArgumentException: Can only configure NSFW (age-restricted) for channels of types TEXT, VOICE, NEWS, FORUM
```
this error occures, when you try to set the slowmode for a voice/stage channel.
If you try to set the nsfw state then the second error comes for stage channels.

Another Note:
> For Forum Channels there doesnt exist "messageTimeout" (got from `getDefaultThreadSlowmode()`) and "tagRequired"

### Code Example for Reproduction Steps

```java
public static void main(String[] args) {
    Guild guild =...;
    guild.createStageChannel("name").setNSFW(true).setSlowmode(100).queue();
    guild.createVoiceChannel("name").setNSFW(true).setSlowmode(100).queue();
    //should both not error, but does.
}
```

### Code for JDABuilder or DefaultShardManagerBuilder used

```java
/
```

### Exception or Error

```yesyes
java.lang.IllegalArgumentException: Can only configure Slowmode for channels of types TEXT, GUILD_NEWS_THREAD, GUILD_PUBLIC_THREAD, GUILD_PRIVATE_THREAD, FORUM
java.lang.IllegalArgumentException: Can only configure NSFW (age-restricted) for channels of types TEXT, VOICE, NEWS, FORUM
```

**Comment by kazuryyx** [comment]
working on it

**Comment by IBPrivate** [comment]
That only fixes half

**Comment by MinnDevelopment** [comment]
Solved in [5.0.0-beta.14](https://github.com/discord-jda/JDA/releases/tag/v5.0.0-beta.14).
